web/satellite/vuetify-poc/src/components/FileCard.vue

```diff
@@ -2,52 +2,52 @@
 // See LICENSE for copying information.
 
 <template>
-    <v-card variant="flat" :border="true" rounded="xlg">
+    <v-card variant="outlined" rounded="lg">
         <div class="h-100 d-flex flex-column justify-space-between">
-            <v-img
-                v-if="objectPreviewUrl && previewType === PreviewType.Image"
-                :src="objectPreviewUrl"
-                alt="preview"
-                height="200px"
-                cover
-                @click="emit('previewClick', item.browserObject)"
-            />
-            <div
-                v-else
-                class="d-flex flex-column justify-center align-center file-icon-container"
-                @click="emit('previewClick', item.browserObject)"
-            >
-                <img
-                    :src="item.typeInfo.icon"
-                    :alt="item.typeInfo.title + 'icon'"
-                    :aria-roledescription="item.typeInfo.title + 'icon'"
-                    height="100"
+            <a role="button" class="h-100" @click="emit('previewClick', item.browserObject)">
+                <v-img
+                    v-if="objectPreviewUrl && previewType === PreviewType.Image"
+                    :src="objectPreviewUrl"
+                    class="bg-light card-preview-img h-100"
+                    alt="preview"
+                    :aspect-ratio="1/1"
+                    cover
+                />
+                <div
+                    v-else
+                    class="d-flex h-100 bg-light flex-column justify-center align-center file-icon-container card-preview-icon"
+                    :aspect-ratio="1/1"
```

Comment:
I'm not sure `aspect-ratio` is am attribute of `div`

Reply:
https://vuetifyjs.com/en/components/aspect-ratios/